client/pdw2pagermon.py

```diff
@@ -0,0 +1,67 @@
+# A program made by Shane (Shaggs) Rees to use PDW's email function into python.
+""" To get Started
+Options -> SMTP / email settings
+Setting set to all messages
+SMTP Host 127.0.0.1
+Port 8826
+To can be set as anything 
+From can be set as anything
+
+Mail options select
+Address, Time, Date, Bitrate, Message
+
+Notification set to messages 
+"""
+import asyncore
+from datetime import datetime
+import email
+from flexcodes import CODES
+from clint.textui import puts, colored
+import serial
+import time
+import smtpd
+import requests
+frag = " "
+def apost(flexcode, msg, when):
+		headers = {
+			'X-Requested-With': 'XMLHttpRequest',
+			"apikey": "", #pagermon APIKey
```

Comment:
I didn't say it doesn't work, more to standardize considering we've added the user agent to reader.js in https://github.com/davidmckenzie/pagermon/pull/173/files and i just dropped one in for the powershell client. 

Just makes it easier to see how people are submitting messages when supporting them as per @marshyonline in https://github.com/davidmckenzie/pagermon/pull/173/files
> This will make filtering out API requests from decoders a simple task

Reply:
got ya my bad. yeah looks ok